R/followers.R

```diff
@@ -7,12 +7,8 @@ friends <- function(user_id, n=NULL, ...) {
 }
 
 ffBase <- function(type, user_id, n=NULL, ...) {
-  if (can_access_other_account(user_id)) {
-    params <- parseUsers(user_id)
-    doCursorAPICall(paste(type, 'ids', sep='/'), 'ids', num=n, params=params, method='GET', ...)
-  } else {
-    warning("Can not lookup relationships for user id ", user_id)
-  }
+  params <- parseUsers(user_id)
```

Comment:
In hindsight, it's probably enough to simply make the warning message a little more descriptive. I'll push something in a bit.